packages/validator/src/options.ts

```diff
@@ -0,0 +1,16 @@
+import {getDefaultGraffiti} from "@chainsafe/lodestar-utils";
+
+export interface IAccountPathOptions {
+  keystoresDir?: string;
+  secretsDir?: string;
+}
+
+export interface IValidatorOptions {
+  graffiti: string;
+  account: IAccountPathOptions;
+}
+
+export const defaultOptions: IValidatorOptions = {
+  graffiti: getDefaultGraffiti(),
```

Comment:
lodestar-validator runs on the browser too. It must not do anything that interacts with the filesystem. Move this logic back to the CLI